server/channels/app/platform/web_conn.go

```diff
@@ -235,9 +239,11 @@ func (ps *PlatformService) NewWebConn(cfg *WebConnConfig, suite SuiteIFace, runn
 	wc.SetSessionToken(cfg.Session.Token)
 	wc.SetSessionExpiresAt(cfg.Session.ExpiresAt)
 	wc.SetConnectionID(cfg.ConnectionID)
-	wc.SetActiveChannelID("")
-	wc.SetActiveTeamID("")
-	wc.SetActiveThreadChannelID("")
+	// <> means unset. This is to differentiate from empty value.
+	// Because we need to support mobile clients where the value might be unset.
```

Comment:
Could you help me understand in which instances a client may be using the empty string? Is it for the active thread only so far or there's more?

Reply:
Active thread only so far. There might be 2 active threads at the same time which I am planning to address.